cl/api/templates/visualizations-api-docs-vlatest.html

```diff
@@ -75,6 +75,18 @@ <h2 id="editing">Editing and Deleting Visualizations</h2>
   <p>Similar commands can be used to update other non-readonly fields.</p>
   <p>To soft-delete a visualization, flip the <code>deleted</code> field to <code>True</code>. To hard-delete, send an HTTP <code>DELETE</code> request.
   </p>
+
+  <h2 id="deprecation-notice">DEPRECATION NOTICE</h2>
```

Comment:
I don't think we need to use caps lock, so:

```suggestion
  <h2 id="deprecation-notice">Deprecation Notice</h2>
```

Reply:
The changes are ready 👍